archive_screenshots.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I recommend always adding this to reduce footgun potential:

```suggestion
#!/usr/bin/env bash
set -euo pipefail
```